invenio_accounts/config.py

```diff
@@ -334,3 +334,6 @@
 
 .. note:: Used for both form help text and for form validation error.
 """
+
+ACCOUNTS_DEFAULT_VERIFIED_STATUS = False
```

Comment:
We discussed having a default for the verified status of users.

Some instances might prefer to have users verified by default.

Reply:
```suggestion
ACCOUNTS_DEFAULT_USERS_VERIFIED = True
```

I would enable it by default as it might be relevant only for zenodo